src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,64 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+import duke.utils.DateUtil;
+
+/**
+ * Event
+ */
+public class Event extends Task {
+
+    private LocalDateTime from;
+    private LocalDateTime to;
```

Comment:
maybe can change this one to "startDate"/"endDate" so it's a little bit clearer? but it looks good anyways!

Reply:
I second this!

Reply:
will certainly do.